src/test/java/lotto/common/utill/InputParserTest.java

```diff
@@ -0,0 +1,89 @@
+package lotto.common.utill;
+
+import static lotto.common.constant.ErrorMessage.*;
+import static lotto.common.utill.InputParser.*;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+import static org.assertj.core.api.AssertionsForInterfaceTypes.*;
+
+import java.util.List;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Nested;
+import org.junit.jupiter.api.Test;
+
+class InputParserTest {
+
+    @Nested
+    class ParseToIntTest {
+        @DisplayName("String 입력값을 Int 값으로 변환한다.")
```

Comment:
테스트명을 구체적으로 작성하니 어떤 테스트인지 훨씬 파악하기 쉬워졌네요!
`@Nested`로 묶으니 어떤 기능에 대한 테스트들인지 구분하기도 쉬워졌네요  👍🏻

Nested에도 DisplayName를 달아서 어떤 기능 테스트인지 작성할 수 있어요 ㅎㅎ

```suggestion
    @Nested
    @DisplayName("String 입력값을 Int 값으로 변환할 때")
    class ParseToIntTest {
        @DisplayName("정상적으로 변환한다.")
```